src/frontend/src/common/types/bioinformatics.d.ts

```diff
@@ -3,14 +3,19 @@ interface BioinformaticsType {
   type: "BioinformaticsType";
 }
 
+interface GISAID {
```

Comment:
GISAID response is now an object instaed of a string

Reply:
I think it would be better to just add two keys, `gisaid_status` and `gisaid_id`, instead of embedding an object just for two keys.

Reply:
Ah I was thinking an object here works out pretty well , since it naturally namespaces `gisaid_`. What do you like about `gisaid_status` and `gisaid_id`?

Reply:
It keeps the response a flat object, which I find easier to reason about and perform transformations on, or write generic functions involving them.

Reply:
Ah key transformation and flattening 💡  

Meta went down the API response key transformation path 4 years ago that recursively transform key from snake_case to camelCase AND flatting the response, the engineers found it pretty confusing and bug prone, especially around typings, debugging, and namespace collision 

So we switched over to just having the single source of truth being the API response, and accepting snake_case for JSON responses, and keeping the nested structure. Then we type each response interface individually composed of entity interfaces. This made things a lot easier to reason and more predictable. Single Cell Team is doing the same thing too!

This also means that we no longer need the generic function for key transformation, which is one less layer of code to deal with, and we get to keep things simple!

Reply:
I meant more data transformations, but :shrug: